### PR TITLE
Left Padding removed in Settings Page Tabs

### LINKED ISSUE #7100: Remove 8px Left Padding in Settings Page Tabs
# Current behavior

There is an unexpected 8px left padding when there are multiple emails/calendar accounts connected to Twenty

CleanShot 2024-09-17 at 17 32 45

# Desired Behavior

The gap is **necessary** for the record detail page. **Keep it as today**

CleanShot 2024-09-17 at 17 35 20

But it should be removed for the Settings page Tabs:

CleanShot 2024-09-17 at 17 36 48

# Figma

App -> https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=32530-139410&node-type=frame&t=vjg6Ofev4bkaijG1-11

Settings -> https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=34851-511709&node-type=frame&t=vjg6Ofev4bkaijG1-11

### PR DESCRIPTION
**Pull request by shyamsundertard**
Fix:  #7100 

The `TabList` component, located in [Tablist](packages/twenty-front/src/modules/ui/layout/tab/components/TabList.tsx), wraps the Tabs and defines the padding, and is used in multiple places. The left padding for the Emails and Calendars sections of the Accounts in Settings has been removed ( list appear when there are multiple connected accounts ). However, the padding on the Record detail page remains unchanged.

To address this, prop of css styles is added to `Tablist`, allowing for the padding of the `TabList` component to be adjusted as required. 

Additional styles can also be applied as per requirements individually for Emails and Calendar section.

Screenshot 2024-10-16 at 5 06 26 AM
Screenshot 2024-10-16 at 5 49 18 AM
Screenshot 2024-10-16 at 6 22 30 AM

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/layout/tab/components/CustomPaddings.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+export const CustomPaddings: React.FC<CustomPaddingsProps> = ({
+  padding,
+  children,
+}) => {
+  const paddingValue = typeof padding === 'number' ? `${padding}px` : padding;
+
+  return (
+    <div
+      className={`custom-padding`}
```
[the remark this answers is not in the document]
Comment (shyamsundertard) [reply packages/twenty-front/src/modules/ui/layout/tab/components/CustomPaddings.tsx:16]: Now, className prop is removed from CustomPaddingsProps

File: packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsCalendarChannelsContainer.tsx [hunk 2]
```diff
@@ -63,10 +64,14 @@ export const SettingsAccountsCalendarChannelsContainer = () => {
     <>
       {tabs.length > 1 && (
         <StyledCalenderContainer>
-          <TabList
-            tabListId={SETTINGS_ACCOUNT_CALENDAR_CHANNELS_TAB_LIST_COMPONENT_ID}
-            tabs={tabs}
-          />
+          <CustomPaddings padding="0">
```
[the remark this answers is not in the document]
Comment (shyamsundertard) [reply packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsCalendarChannelsContainer.tsx:67]: Yes, a separate file can be created in which DEFAULT_PADDING = '0' can be set and DEFAULT_PADDING will be imported and used in place where padding="0" was used.

**Comment by shyamsundertard** [comment]
If there are any changes that have to be made in it please let me know.

**Comment by shyamsundertard** [comment]
@Bonapara can you please review this PR

**Comment by shyamsundertard** [comment]
This is my first PR, and I'm learning as I go.

**Comment by Bonapara** [comment]
Looks good from a design perspective; I'll leave the code to the tech team.

**Review by charlesBochet — APPROVED** [review]
I've found a simpler fix :)

**Pull request merged**